Documentation/ceph-advanced-configuration.md

```diff
@@ -590,3 +591,42 @@ ceph osd pool get replicapool crush_rule
 If the cluster's health was `HEALTH_OK` when we performed this change, immediately, the new rule is applied to the cluster transparently without service disruption.
 
 Exactly the same approach can be used to change from `host` back to `osd`.
+
+## Auto Expansion of OSDs
+
+### Prerequisites
+
+1) A [PVC-based cluster](ceph-cluster-crd.md#pvc-based-cluster) deployed in dynamic provisioning environment with a `storageClassDeviceSet`.
+
+2) Create rook-ceph ToolBox Deployment.
```

Comment:
No need to show the toolbox.yaml on the lines below, a reference to that doc is ok. They probably already have the toolbox running anyway.
```suggestion
2) Create the Rook [Toolbox](ceph-toolbox.md).
```

Reply:
Yes, modified the doc.